en/08_Changelogs/6.0.0.md

```diff
@@ -29,6 +30,242 @@ title: 6.0.0 (unreleased)
+
+See [`PolyCommand`](/developer_guides/cli/polycommand#buildtask) for more details about the `BuildTask` API.
+
+##### Change to names {#cli-buildtask-names}
+
+Some tasks were relying on the FQCN instead of having an explicit name. This means the name used for both the URL and the CLI command were excessively long.
+
+The way these are displayed in the new `sake tasks` list doesn't suit long names, so we have given explicit names to these tasks in order to make them shorter. If you have scripts or cron jobs that reference these you will need to update them to use the new name.
+
+For example, you used to navigate to `/dev/tasks/<OLD_NAME>` or use `sake dev/tasks/<OLD_NAME>`. Now you will need to navigate to `/dev/tasks/<NEW_NAME>` or use `sake tasks:<NEW_NAME>`.
+
+|class|old name|new name|
+|---|---|---|
+|[`ContentReviewEmails`](api:SilverStripe\ContentReview\Tasks\ContentReviewEmails)|`SilverStripe-ContentReview-Tasks-ContentReviewEmails`|`content-review-emails`|
+|[`DeleteAllJobsTask`](api:Symbiote\QueuedJobs\Tasks\DeleteAllJobsTask)|`Symbiote-QueuedJobs-Tasks-DeleteAllJobsTask`|`delete-queued-jobs`|
+|[`MigrateContentToElement`](api:DNADesign\Elemental\Tasks\MigrateContentToElement)|`DNADesign-Elemental-Tasks-MigrateContentToElement`|`elemental-migrate-content`|
+|[`UserFormsColumnCleanTask`](api:SilverStripe\UserForms\Task\UserFormsColumnCleanTask)|`SilverStripe-UserForms-Task-UserFormsColumnCleanTask`|`userforms-column-clean`|
+|[`StaticCacheFullBuildTask`](api:SilverStripe\StaticPublishQueue\Task\StaticCacheFullBuildTask)|`SilverStripe-StaticPublishQueue-Task-StaticCacheFullBuildTask`|`static-cache-full-build`|
+
+#### Changes to `/dev/*` actions {#cli-dev}
+
+With the changes to `sake` come changes to the way `dev/*` actions are handled. Most of these are now subclasses of the new [`DevCommand`](api:SilverStripe\Dev\Command\DevCommand) class which is itself a subclass of `PolyCommand`.
+
+If you have custom actions registered under `DevelopmentAdmin.registered_controllers` you'll need to update the YAML configuration for these. If you want them to be accessible via CLI, you'll also have to update the PHP code.
+
+With the below example, there are two custom actions displayed in the list at `/dev`:
+
+- `/dev/my-action`: intended for use in the browser only, but you'd have to add custom logic in `init()` to disallow its use in CLI until now
+- `/dev/my-other-action`: intended for use both in CLI and in the browser.
+
+For actions that should only be accessible in the browser, you only need to change how these are registered. Move them from `DevelopmentAdmin.registered_controllers` to the new [`DevelopmentAdmin.controllers`](api:SilverStripe\Dev\DevelopmentAdmin->controllers) configuration property.
+
+Controllers added to `DevelopmentAdmin.registered_controllers` can only be accessed via HTTP requests, so you can remove any logic around CLI usage.
```

Comment:
```suggestion
Controllers added to `DevelopmentAdmin.controllers` can only be accessed via HTTP requests, so you can remove any logic around CLI usage.
```

Reply:
Duplicate of https://github.com/silverstripe/developer-docs/pull/571#discussion_r1770699036?